src/main/java/videoshop/customer/CustomerController.java

```diff
@@ -43,7 +45,7 @@
 	// Siehe außerdem videoshop.model.validation.RegistrationForm
 	// Lektüre: http://docs.spring.io/spring/docs/current/spring-framework-reference/html/validation.html
 	@PostMapping("/register")
-	String registerNew(@Valid RegistrationForm form, Errors result) {
+	String registerNew(@Valid @ModelAttribute("form") RegistrationForm form, BindingResult bindingResult, Errors result) {

```

Comment:
Adding `BindingResult` shouldn't be necessary as that's what `Errors` actually already is. Also, it's not actually used in the method below.

```suggestion
	String registerNew(@Valid @ModelAttribute("form") RegistrationForm form, Errors result) {
```